libs/storage/storage.cpp

```diff
@@ -75,22 +75,11 @@ snorfs::File *getFile(String filename) {
     }
     currFilename = filename;
     incrRC(currFilename);
-    currFile = filename == NULL ? NULL : st->fs.open(filename->data);
+    // TODO: fix UTF8 encoding
+    currFile = filename == NULL ? NULL : st->fs.open(filename->getUTF8Data());
     return currFile;
 }
 
-/** 
-* Append string data to a new or existing file. 
-* @param filename name of the file, eg: "log.txt"
-*/
-//% parts="storage" 
-//% blockId="storage_append" block="append file $filename with $data"
-void append(String filename, String data) {
-    auto f = getFile(filename);
-    if (NULL == f) return;
-    f->append(data->data, data->length);
-}
-
 /** 
```

Comment:
Don't remove these - just use getUtf8data and getutf8size - they are close enough to real UTF to be used in files